WNPRC_Purchasing/resources/queries/ehr_purchasing/purchasingRequestsOverview.sql

```diff
@@ -1,11 +1,17 @@
-SELECT pr.account,
-       pr.qcStatus         AS requestStatus,
+SELECT pr.rowId,
+       pr.vendorId,
+       pr.account,
+       pr.otherAcctAndInves,
+       pr.qcState        AS requestStatus,
        pr.created          AS requestDate,
        pr.createdBy        AS requestor,
-       sum(items.unitCost) AS totalCost
+       round(sum(items.quantity * items.unitCost), 2) AS totalCost
 FROM ehr_purchasing.purchasingRequests pr
 LEFT JOIN ehr_purchasing.lineItems items ON pr.requestId = items.requestId
```

Comment:
This will be more efficient if you do a subselect on lineItems that groups by just requestId, and then join that with purchasingRequests. Let me know if you want to talk about why this is, but the basic idea is that it reduces the complexity of the GROUP BY

Reply:
Yup that makes sense, my query was pretty slow as well, this helped! Thank you!